### DISCUSSION TITLE
Number of particles formula in particle dissolution model

### DISCUSSION THREAD

**Discussion by moPellowe** [Open-Systems-Pharmacology/Forum#1126] [Questions & Problems] [answered]
Can someone please explain the "Number of particles" formula within the Particle Dissolution model (within the ParticleBin event group)? I don't understand why MW is there, and the units don't seem to line up.

DrugMass [g] / DrugDensity [g/L] * MW [g/mol] * Number_Of_Particles_Factor [1/L]

Without MW, the formula makes more sense to me.

DrugVolume = DrugMass / DrugDensity
Number_Of_Particles_Factor : the number of particles for that molecule per volume unit
Number of particles = DrugVolume * Number_Of_Particles_Factor

![image](https://user-images.githubusercontent.com/57540473/153195659-454a4f23-9c4d-47c3-8df7-d62c270ccb0d.png)

**Comment by Yuri05** [comment, marked as answer]
DrugMass here is an **amount** => [µmol]
So in the internal units it is
DrugMass [µmol] / DrugDensity [kg/L] * MW [kg/µmol] * Number_Of_Particles_Factor [1/L] => Dimensionless

**Reply by moPellowe** [reply]
Ah, I see. Thank you. :)
